ansible/roles/test/files/ptftests/sonic.py

```diff
@@ -166,14 +166,23 @@ def run(self):
         log_data = {}
 
         self.log('Collecting logs')
-        log_lines = self.do_cmd("sudo cat "
-                                "/var/log/syslog{,.1} "
-                                "/var/log/teamd.log{,.1} "
-                                "/var/log/frr/bgpd.log "
-                                "/var/log/frr/zebra.log").split('\n')
+        log_files = [
+            "/var/log/syslog{,.1}",
+            "/var/log/teamd.log{,.1}",
```

Comment:
Since there is now logic to stop early when the logs being parsed are older than `start_time`, these lines need to be `/var/log/syslog{.1,}` and `/var/log/teamd.log{.1,}`. Otherwise, the order in which logs are printed will be messed up.

Reply:
Fixed